Cache/Hashing.lean

```diff
@@ -118,7 +128,13 @@ partial def getHash (filePath : FilePath) (visited : Std.HashSet FilePath := ∅
       modify fun stt => { stt with cache := stt.cache.insert filePath none }
       return none
     let content ← IO.FS.readFile fixedPath
-    let fileImports := getFileImports content (← getPackageDirs)
+    let fileImports' ← getFileImports content filePath.toString
+    -- TODO: This line is only here for backwards compatibility: cache still takes
+    -- keys of the form `Mathlib/Init.lean`, `Aesop/Build.lean` instead of module
```

Comment:
What do you mean by "cache takes"; are you referring to the command line arguments? Or some internal function / data structure?

Reply:
both in fact

Reply:
updated comment